ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -273,6 +273,8 @@ def _fetch_tests_to_skip(self):
             self._test_suites_to_skip = []
             return
 
+        self._test_suites_to_skip = []
```

Comment:
Moving this up prevents us from potentially crashing when fetching skippable tests fails.

Running more tests than expected (the consequence of failing to fetch skippable tests) is preferable to the recorder hitting an exception and preventing tests from running.